package.json

```diff
@@ -32,8 +32,8 @@
     "styled-components": "^4.4.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
+    "build": "SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts build",
```

Comment:
I looked at this a touch and think this is okay- just will want to make sure that this doesn't goof up any of the deployment of the app. I don't think it will, just want to double check.

Reply:
Ah, looks like this is going to cause a problem with building the app: https://app.netlify.com/sites/whidbey-herbal/deploys/6264cc2b6af7d60008c68a68

Also, can you see the above link? If not I'll wanna get you access to netlify!